workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/notification-templates/new-app-submitted/en-US.html

```diff
@@ -38,6 +38,9 @@
 				padding: 0.5rem 1rem;
 				text-decoration: none;
 			}
+			.submitted-app-class-321{
+				margin-left: 12px;
+			}
```

Comment:
Spaces

@ricardomariz It is better to move this code to another Pull Request

```suggestion

			.submitted-app-class-321 {
				margin-left: 12px;
			}
			
```